dask/tests/test_utils.py

```diff
@@ -49,16 +49,17 @@ def test_dispatch():
     foo.register(float, lambda a: a - 1)
     foo.register(tuple, lambda a: tuple(foo(i) for i in a))
     foo.register(object, lambda a: a)
+    foo_int = foo.dispatch(int)
 
     class Bar(object):
         pass
     b = Bar()
     assert foo(1) == 2
+    assert foo_int(1) == 2
```

Comment:
Preference to do move the dispatch call to where it's used in the test. `assert foo.dispatch(int)(1) == 2`